pkg/test/collector_intermediate_test.go

```diff
@@ -219,8 +219,8 @@ func testCollectorToIntermediate(t *testing.T, address net.Addr, isIPv6 bool) {
 
 }
 
-func copyFlowKeyRecordMap(key intermediate.FlowKey, aggregationFlowRecord intermediate.AggregationFlowRecord) error {
-	flowKeyRecordMap[key] = aggregationFlowRecord
+func copyFlowKeyRecordMap(key intermediate.FlowKey, aggregationFlowRecord *intermediate.AggregationFlowRecord) error {
+	flowKeyRecordMap[key] = *aggregationFlowRecord
```

Comment:
Since it is only for testing, I copied values from flowKeyRecordMap instead of the whole map. That is also a reminder for users to not copy pointers when calling `ForAllRecordsDo`

Reply:
Got it, thanks.